### PR TITLE
init

### PR DESCRIPTION
**Pull request by michaelwittig**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by michaelwittig** [comment]
test

**Comment by michaelwittig** [comment]
test

**Review by andreaswittig — APPROVED** [review]

File: module.yml [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+      - !GetAtt 'DBCluster.Endpoint.Address'
+      TTL: '60'
+      Type: CNAME
+  SecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: !Ref 'AWS::StackName'
+      SecurityGroupIngress:
+      - IpProtocol: tcp
+        FromPort: 3306
```
Comment (andreaswittig) [inline module.yml]: Isn't the default port for PostgreSQL `5432`?

**Review by andreaswittig — CHANGES_REQUESTED** [review]

File: module.yml (hunk 1 above)
Comment (michaelwittig) [reply module.yml]: good catch

**Pull request merged**